packages/builder/builder-webpack-provider/src/core/createCompiler.ts

```diff
@@ -28,16 +23,7 @@ export async function createCompiler({
 
   let isFirstCompile = true;
 
-  compiler.hooks.done.tap('done', async (stats: unknown) => {
-    const { message, level } = formatStats(stats as Stats);
```

Comment:
So formatWebpackMessages is no longer called?

Reply:
I think so, or maybe it can be one transformer of the friendly errors plugin?

Reply:
okey, I will work on it.